OpenRA.Game/Player.cs

```diff
@@ -193,7 +191,7 @@ public void OnScriptBind(ScriptContext context)
 		public LuaValue Equals(LuaRuntime runtime, LuaValue left, LuaValue right)
 		{
 			Player a, b;
-			if (!left.TryGetClrValue<Player>(out a) || !right.TryGetClrValue<Player>(out b))
+			if (!left.TryGetClrValue(out a) || !right.TryGetClrValue(out b))
```

Comment:
Those were marked as redundant, because the expected type is already specified on the line above: `Player a, b;`.